cocos/audio/assets/clip.ts

```diff
@@ -132,8 +132,8 @@ export class AudioClip extends Asset {
 }
 
 export namespace AudioClip {
-    export type PlayingState = typeof PlayingState;
-    export type AudioType = typeof AudioType;
+    export type PlayingState = EnumAlias<typeof PlayingState>;
```

Comment:
I think typeof is not needed anymore, isn't it ?

Reply:
It's necessary. Otherwise tsc will say 'Type alias circular reference itself.'

Reply:
> Will all those declarations be stripped out in the JS file
These declarations would not be emitted in output JS.